src/main/java/calculator/exception/ValidateNumber.java

```diff
@@ -0,0 +1,9 @@
+package calculator.exception;
+
+public class ValidateNumber extends IllegalArgumentException{
```

Comment:
```suggestion
public class ValidateNumber extends IllegalArgumentException {
```

혹시 IntelliJ를 사용하신다면 단축키를 찾아보셔도 좋을 것 같아요 👀